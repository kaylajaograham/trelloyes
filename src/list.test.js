import React from 'react';
import ReactDOM from 'react-dom';
import List from './list';
import renderer from 'react-test-renderer';


describe('list component', () => {
    
    // smoke test
    it("renders without crashing", () => {
        const div = document.createElement("div");
        ReactDOM.render(<List header="Test Header" cards={[]} />, div);
        ReactDOM.unmountComponentAtNode(div);
    });

    it("renders the UI as expected", () => {
        const tree = renderer
          .create(<List header="Test Header" cards={[]} />)
          .toJSON();
        expect(tree).toMatchSnapshot();  
      });
})