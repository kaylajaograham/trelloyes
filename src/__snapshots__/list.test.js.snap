// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`list component renders the UI as expected 1`] = `
<section
  className="List"
>
  <header
    className="List-header"
  >
    <h2>
      Test Header
    </h2>
  </header>
  <div
    className="List-cards"
  >
    <button
      onClick={[Function]}
      type="button"
    >
      Add Random Card
    </button>
  </div>
</section>
`;
